### ISSUE TITLE
False example value not working for boolean fields

### ISSUE THREAD

**Issue by nakedible-p** [Rebilly/ReDoc#111]
Referencing #109, it seems that `example: false` still does not work.

```js
      field:
        type: boolean
        default: false
        example: false
```
Produces `Default: false` nicely in the documentation, but `field: true` in the example message.

**Comment by RomanGotsiy** [comment]
@nakedible-p, yep, my fault :) Thanks for reporting.
Will be fixed in the next release

**Comment by nakedible-p** [comment]
Great! Excellent responsiveness :+1:

**Comment by RomanGotsiy** [comment]
Fixed in the latest minor release: v1.3.3
